modules/silicon_design/scripts/build/images/provision.sh

```diff
@@ -60,3 +61,12 @@ cp ${PROVISION_DIR}/papermill-launcher /usr/local/bin/
 chmod +x /usr/local/bin/papermill-launcher
 
 echo "DaisySuccess: done"
+
+echo "Cloning CFU-Playground repository"
+git clone -b ${CFU_PLAYGROUND_VERSION} https://github.com/google/CFU-Playground.git /CFU-Playground
```

Comment:
right here

Reply:
oh I see, I didn't realize that your branch was already in the main repo (I thought it would be in a fork)